pkg/ccl/crosscluster/replicationutils/utils.go

```diff
@@ -237,6 +239,25 @@ func InvestigateFingerprints(
 	return nil
 }
 
+func GetDbAndSchemaPrefixesForTable(
+	ctx context.Context, txn *kv.Txn, descriptors *descs.Collection, tableDesc descpb.TableDescriptor,
+) (string, error) {
+	var prefixes string
+	byIDGetter := descriptors.ByID(txn).WithoutNonPublic().Get()
+	dbDesc, err := byIDGetter.Database(ctx, tableDesc.GetParentID())
+	if err != nil {
+		return "", err
+	}
+
+	scDesc, err := byIDGetter.Schema(ctx, tableDesc.GetUnexposedParentSchemaID())
+	if err != nil {
+		return "", err
+	}
+
+	prefixes = fmt.Sprintf("%s.%s", dbDesc.GetName(), scDesc.GetName())
```

Comment:
_Somewhere_ I think we will want to escape each individual part of the name using `lexbase.EscapeSQLIdent`. Doing that here might make the parsing you are doing later harder.  See my comment on the protobuf for another idea.